Utils/azuretools-core/pom.xml

```diff
@@ -303,4 +310,12 @@
             <artifactId>jtwig-core</artifactId>
         </dependency>
     </dependencies>
+
+    <repositories>
```

Comment:
add todo comment since it will be removed after mysql sdk releases

Reply:
will remove once got a ultimate version MySQL SDK.